maya/mel/assign-pbr-textures.mel

```diff
@@ -0,0 +1,43 @@
+{
+    $sl = {""};
+    $sl = `ls -sl`;
```

Comment:
In MEL, the guideline is to mention the type explicitly, e.g. `string $sel[] = `...